contracts/ocr2/src/contract.rs

```diff
@@ -1472,8 +1499,8 @@ fn calculate_reimbursement(
     signature_count: usize,
 ) -> Uint128 {
     let signature_count = signature_count as u64;
-    let gas_per_signature = config.gas_per_signature.unwrap_or(17_000);
-    let gas_base = config.gas_base.unwrap_or(84_000);
+    let gas_per_signature = config.gas_per_signature.unwrap_or(4_096);
+    let gas_base = config.gas_base.unwrap_or(146_000);
     let gas_adjustment = Decimal::percent(u64::from(config.gas_adjustment.unwrap_or(140)));
```

Comment:
We synced offline on the topic of "On chain 1.4x multiplier produces matching values to 1.5x offchain"...

Let's add a descriptive in-line comment here explaining why are the values different, and where is the discrepancy.